packages/ui/package.json

```diff
@@ -202,6 +203,7 @@
     "mersenne-twister": "^1.1.0",
     "react-contenteditable": "^3.3.7",
     "react-feather": "^2.0.10",
+    "react-lottie": "^1.2.10",
```

Comment:
I believe this package is not used; if so please remove it as well as its types pkg.